src/test/isolation2/sql/copy_interrupt.sql

```diff
@@ -0,0 +1,15 @@
+!\retcode gpconfig -c client_connection_check_interval -v 20s;
+!\retcode gpstop -u;
+
+0: CREATE TABLE copy_interrupt_table(a int);
+0&: COPY copy_interrupt_table FROM PROGRAM 'while true; do echo 1; sleep 1; done | cat -';
+! sleep 10;
+
+0t:
+! sleep 40;
+
+SELECT pg_terminate_backend(pid)
```

Comment:
In case the test fails, backend will just hang forever. Better terminate it after the main part of the test is done. If patch works correctly, there shouldn't be any backends to terminate, so there will be zero rows in the output. I will add a comment about this to the test.